client/src/components/ProductDetails/Carousel.jsx

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Carousel extends React.Component {
```

Comment:
I think this could benefit from more comments

Reply:
Added more comments throughout the file! Thank you